### PR TITLE
created log return indicator and tests

### PR DESCRIPTION
**Pull request by chutes**
created log return indicator and tests

### CODE REVIEW

**Comment by mchandschuh** [comment]
I can produce some  test data for you, I'll push it tonight.

**Comment by chutes** [comment]
Thanks

**Comment by mchandschuh** [comment]
https://github.com/QuantConnect/Lean/commit/52531aa216f5fe22e86b8a44f6831303c5e35d16 adds LOGR14 test data from freestockcharts.com using formula: LOG(c/c14)

File: Indicators/LogReturn.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+
+namespace QuantConnect.Indicators
+{
+    /// <summary>
+    ///     Represents the LogReturn indicator (LOGR)
```
Comment (mchandschuh) [inline Indicators/LogReturn.cs]: Can we add a little bit more here, maybe include the equation used and why someone would use it. Here's a good [read]https://quantivity.wordpress.com/2011/02/21/why-log-returns/) to start with.

**Comment by mchandschuh** [comment]
@chutes - thanks for the great contribution! Code/doc/tests look great.

**Pull request merged**